Makefile

```
@@ -234,26 +242,26 @@ install-pipelines:
 # local testing
 KIND_CLUSTER_NAME ?= "meteor-cnbi"
 
-.PHONY: kind-start
-kind-start:
+.PHONY: kind-create
+kind-create:
```

I think this should depend on `kind`, right?

```suggestion
kind-create: kind
```